### PR TITLE
DOCSUP-15458: Document the SQL-statement ALTER TABLE ... MODIFY COMMENT

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #29264

### CODE REVIEW

File: docs/en/sql-reference/statements/alter/comment.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+)
+ENGINE = Memory()
+COMMENT 'The temporary table';
+```
+
+Modifying the table comment:
+
+``` sql
+ALTER TABLE table_with_comment MODIFY COMMENT 'new comment on a table';
+SELECT comment FROM system.tables WHERE database = currentDatabase() and name = 'table_with_comment';
```
Comment (olgarev) [inline docs/en/sql-reference/statements/alter/comment.md:34]:
```suggestion
SELECT comment FROM system.tables WHERE database = currentDatabase() AND name = 'table_with_comment';
```

File: docs/en/sql-reference/statements/alter/comment.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+┌─comment────────────────┐
+│ new comment on a table │
+└────────────────────────┘
+```
+
+Removing the table comment:
+
+``` sql
+ALTER TABLE table_with_comment MODIFY COMMENT '';
+SELECT comment FROM system.tables WHERE database = currentDatabase() and name = 'table_with_comment';
```
Comment (olgarev) [inline docs/en/sql-reference/statements/alter/comment.md:49]:
```suggestion
SELECT comment FROM system.tables WHERE database = currentDatabase() AND name = 'table_with_comment';
```

File: docs/en/sql-reference/statements/alter/index.md [hunk 3]
```diff
@@ -32,6 +32,8 @@ These `ALTER` statements modify entities related to role-based access control:
 -   [ROW POLICY](../../../sql-reference/statements/alter/row-policy.md)
 -   [SETTINGS PROFILE](../../../sql-reference/statements/alter/settings-profile.md)
 
+[ALTER TABLE ... MODIFY COMMENT](../../../sql-reference/statements/alter/comment.md) statement adds, modifies, or removes comments to table.
```
Comment (olgarev) [inline docs/en/sql-reference/statements/alter/index.md:35]:
```suggestion
[ALTER TABLE ... MODIFY COMMENT](../../../sql-reference/statements/alter/comment.md) statement adds, modifies, or removes comments to the table.
```

**Comment by sevirov** [comment]
@alesapin Please, review this PR.

**Review by alesapin — APPROVED** [review]

**Pull request merged**